crates/typst-library/src/model/list.rs

```diff
@@ -37,24 +37,24 @@ use crate::text::TextElem;
 /// )
 /// ```
 ///
-/// # Syntax
-/// This functions also has dedicated syntax: Start a line with a hyphen,
-/// followed by a space to create a list item. A list item can contain multiple
-/// paragraphs and other block-level content. All content that is indented
-/// more than an item's marker becomes part of that item.
+/// # 構文
+/// この関数には専用の構文もあります。
+/// 行頭にハイフンとスペースを置くことでリスト項目を作成します。
+/// リスト項目には複数の段落や、他のブロックレベルコンテンツを含めることができます。
+/// リスト項目の記号よりも深く字下げされた全てのコンテンツは、そのリスト項目の一部になります。
 #[elem(scope, title = "Bullet List", Show)]
 pub struct ListElem {
-    /// Defines the default [spacing]($list.spacing) of the list. If it is
-    /// `{false}`, the items are spaced apart with
-    /// [paragraph spacing]($par.spacing). If it is `{true}`, they use
-    /// [paragraph leading]($par.leading) instead. This makes the list more
-    /// compact, which can look better if the items are short.
+    /// リストのデフォルトの[spacing]($list.spacing)を定義します。
+    /// これが`{false}`の場合、 項目の間隔は[paragraph spacing]($par.spacing)によって決まります。
+    /// `{true}`の場合、代わりに[paragraph leading]($par.leading)が使用されます。
+    /// これによりリストがよりコンパクトになり、
+    /// 各項目が短い場合に見栄えが良くなります。
     ///
-    /// In markup mode, the value of this parameter is determined based on
-    /// whether items are separated with a blank line. If items directly follow
-    /// each other, this is set to `{true}`; if items are separated by a blank
-    /// line, this is set to `{false}`. The markup-defined tightness cannot be
-    /// overridden with set rules.
+    /// マークアップモードでは、
+    /// この引数の値は項目が空行で区切られているかどうかに基づいて決定されます。
+    /// 項目間に空行がなく連続している場合、この値は`{true}`に設定されますが、
+    /// 項目間が空行で区切られている場合は`{false}`に設定されます。
+    /// マークアップで定義された間隔はsetルールで上書きすることは出来ません。
```

Comment:
【提案】「出来る」→「できる」
どちらでも問題はありませんが、漢字だとフォーマルな感じが出てしまうため、ですます調だとひらがなの方が自然かと思います。
```suggestion
    /// マークアップで定義された間隔はsetルールで上書きすることはできません。
```